spec/authentication.yml

```diff
@@ -0,0 +1,823 @@
+          2. Base64-encoding the authentication string.
+          3. Prefixing the authentication string with the scheme: `Basic `
+             (note the required space).
+          4. Providing the result as the value of the `Authorization` HTTP header:
+             `Authorization: Basic <authentication string>`.
+
+          Your HTTP/REST client probably provides HTTP basic authentication support."
+        operationId: "getAPIKeyViaLDAP"
+        parameters:
+        - name: "service_id"
+          in: "path"
+          required: true
+          description: "URL-encoded service ID of the authenticator"
+          schema:
+            type: string
+          example: "authn"
+        - name: "account"
+          in: "path"
+          required: true
+          description: "Organization account name"
+          schema:
+            $ref: 'openapi.yml#/components/schemas/AccountName'
+
+        responses:
+          "200":
+            $ref: 'openapi.yml#/components/responses/ApiKey'
+          "400":
+            $ref: 'openapi.yml#/components/responses/BadRequest'
+          "401":
+            $ref: 'openapi.yml#/components/responses/UnauthorizedError'
+          "404":
+            $ref: 'openapi.yml#/components/responses/ResourceNotFound'
+          "422":
+            $ref: 'openapi.yml#/components/responses/UnprocessableEntity'
+          "500":
+            $ref: 'openapi.yml#/components/responses/InternalServerError'
+
+        security:
+          - basicAuth: []
+
+    DefaultAuthenticate:
+      post:
+        tags:
+        - "authentication"
+        summary: "Gets a short-lived access token, which is required in the header
+          of most subsequent API requests."
+        description: "A client can obtain an access token by presenting a valid
+          login name and API key.
+
+          The access token is used to communicate to the REST API that the bearer
+          of the token has been authorized to access the API and perform specific
+          actions specified by the scope that was granted during authorization.
+
+          The `login` must be URL encoded. For example, `alice@devops` must be
+          encoded as `alice%40devops`.
+
+          The `service_id`, if given, must be URL encoded. For example,
+          `prod/gke` must be encoded as `prod%2Fgke`.
+
+          For host authentication, the `login` is the host ID with the prefix
+          `host/`. For example, the host webserver would login as `host/webserver`,
+          and would be encoded as `host%2Fwebserver`.
+
+          For API usage, the base64-encoded access token is ordinarily passed as an HTTP
+          Authorization header as `Authorization: Token token=<base64-encoded token>`.
+
+          This is the default authentication endpoint only. See other endpoints for
+          details on authenticating to Conjur using another method, e.g. for
+          applications running in Azure or Kubernetes."
+        operationId: "getAccessToken"
```

Comment:
In my PR I ended up using `show` in place of get because of feedback from Jonah. I think it would be best if we stay consistent across the spec. Do you have any thoughts on which verb we should use?

Reply:
this isn't retrieving an existing resource, though - the server is generating a new access token and returning it. @jonahx wdyt?

Reply:
Okay, I agree with this, don't think we need to be using show here.